scripts/ci_mingw64_geany.sh

```diff
@@ -70,19 +70,12 @@ CFLAGS="\
 ARCH="x86_64"
 MINGW_ARCH="mingw64"
 HOST="x86_64-w64-mingw32"
-export CC="/usr/bin/${HOST}-gcc"
-export CPP="/usr/bin/${HOST}-cpp"
-export CXX="/usr/bin/${HOST}-g++"
-export AR="/usr/bin/${HOST}-ar"
-export STRIP="/usr/bin/${HOST}-strip"
-export WINDRES="/usr/bin/${HOST}-windres"
-export CFLAGS="-I/windows/${MINGW_ARCH}/include/ ${CFLAGS}"
```

Comment:
All those variables are not necessary.

I think I carried them over from older build scripts. It builds fine also without those though.